source/module_hamilt_lcao/module_gint/gint_k.h

```diff
@@ -13,7 +13,16 @@
 
 class Gint_k : public Gint {
   public:
-    ~Gint_k() { destroy_pvpR(); }
+    ~Gint_k()
+    {
+        destroy_pvpR();
+    }
+
+    /// @brief move operator for the next ESolver to directly use its infomation
+    /// @param rhs 
+    /// @return *this
+    Gint_k& operator=(Gint_k&& rhs);
```

Comment:
I would suggest removing this line. We don't want operator unless necessary.

Reply:
Move constructors are necessary in my new code. We've talked about this at the meeting on July 2nd (Tue.)